packages/frontend/pages/index.vue

```diff
@@ -18,57 +13,89 @@ GNU Affero General Public License for more details.
-            :right-label="'New Group'"
-        />
-    
-        <!-- <div>Create a Single Asset:</div> -->
-        <div id="create_record"><FormsCreateDevice/></div>
-        <!-- <CreateDevice/> -->
-        <!-- <div>Or, if you want to create a group of keys:</div> -->
-        <!-- <div></div> -->
-       
-        <div id="create_group" style="display:none"><FormsCreateContainer/></div>
- 
- 
-        <h5 class="my-4">
-            The Global Open Source Quality Assurance System (GOSQAS) provides the opportunity to create truthful, transparent documentation via our open source provenance tracker, Global Distributed Tracking (GDT). Designed specifically for low-resource settings, GDT allows anyone with simple website access to securely record their project data. Our development philosophy of “Trust Through Transparency” (TTT) encourages better global communication within diverse communities- to include humanitarian response, open source hardware, and scientific research.
-        </h5>
- 
- 
+    <div class="container-fluid">
+
+        <div class="row" id="first-row">
+            <div class="col-12 col-md-7" id="first-row-col">
+                <div class="row"> <h1>Trust and transparancy when you need it most.</h1> </div>
+                <div class="row" style=" margin-bottom: 60px; margin-top:20px; display:inline-flex">
+                    <form style="margin-bottom: 20px; width:40%; width: 190px; padding-right: 0px;" @submit.prevent="trackingForm">
+                        <button-component class="button" id="trackButton" buttonText="View Record" type="submit" style="opacity:100;"
+                            padding="18px 22px"></button-component>
+                    </form>
+                    <div style="margin-bottom: 20px; width: 60%; width: 230px;" >
+                        <button-component class="button" buttonText="Create Record" backgroundColor="#CCECFD" onclick="window.location.href='/gdt'"
+                            borderColor="#CCECFD" color="#1E2019" padding="18px 22px" margin="0px 0px 0px 0px"></button-component>
+                    </div>
+                    <div id="homeTrackAssetDiv" style="visibility: hidden;">
+                        <TrackAsset inputWidth="75%"></TrackAsset>
+                    </div>
+                </div>
+                
+            </div>
+
+        </div>
+
+        <div class="row bg-frost" id="second-row" >
+            <div class="row rowtest">
+                <div class="col test" id="second-row-cols" v-for="item in second_row">
+                    <h3 class="text-iris" id="second-row-cols-h3">{{item.title}}</h3>
+                    <p class="text-eggplant" style="font-weight: 400;">{{ item.descr }}</p>         
+                </div>
+            </div>
+            <div class="col" style="text-align: center; margin-top: 50px;">
+                <button-component class="button" buttonText="About Us" color="#322253" onclick="window.location.href='about'"
```

Comment:
This works but to be Vue-like can you please create a method that uses the vue router to navigate to the page?